src/late_multicellular_stage/editor/action_data/MetaballResizeActionData.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System;
 using Godot;
 
-public class MetaballResizeActionData<TMetaball> : EditorCombinableActionData
+public class MetaballResizeActionData<TMetaball> : EditorCombinableActionData<LateMulticellularSpecies>
```

Comment:
Does this need the context info? Can the metaball edit actions be left without a context?

Reply:
I added that in because I thought we might need it when we start working on the late multicellular editor, but I'll take it out to reduce the number of changed files.

Reply:
That'd be better I think, because I can't foresee needing this for the reason that the metaball actions are related to the metaballs that are placed, their context would always be related to the species that is being edited (I never want to have to refactor the editors to support editing multiple species at once).